.github/workflows/contracts-deploy.yml

```diff
@@ -50,11 +56,14 @@ jobs:
 
       - name: Install binaryen
         shell: bash
+        # yamllint disable rule:line-length
```

Comment:
Do we need the `binaryen` (and `cargo contract` below) if we're using dockerized `cargo contract` in `contracts/scripts/deploy.sh`?

Reply:
Honestly, I tried to not touch this file as much as possible, I don't really know how to test the changes. Perhaps @fbielejec could check the deployments again after all this is merged.

Reply:
I would definitely not touch any workflow files before it's all unified